front/components/spaces/SpaceResourcesList.tsx

```diff
@@ -543,22 +543,17 @@ export const SpaceResourcesList = ({
           }}
         />
       )}
-      {selectedDataSourceView &&
-        selectedDataSourceView.dataSource.connector && (
-          <>
-            <ConnectorPermissionsModal
-              owner={owner}
-              connector={selectedDataSourceView.dataSource.connector}
-              dataSource={selectedDataSourceView.dataSource}
-              isOpen={showConnectorPermissionsModal && !!selectedDataSourceView}
-              onClose={() => {
-                setShowConnectorPermissionsModal(false);
-              }}
-              readOnly={false}
-              isAdmin={isAdmin}
-            />
-          </>
-        )}
+      <ConnectorPermissionsModal
```

Comment:
is it really needed to render this if there's no connector/datasource ? it complexifies the code a lot and it seems ConnectorPermissionsModal can't do a lot if we don't have data to display ?

Reply:
You right, let's not render it here. Good catch thx 🙏🏼